backends/edk2-compat/edk2-svc-validate.c

```diff
@@ -604,7 +604,6 @@ int parseX509(void **x509, const unsigned char *certBuf, size_t buflen)
 			prlog(PR_ERR, "ERROR: Failed to parse x509 in DER and file is not in PEM\n");
 			return CERT_FAIL;
 		}
-		printf("WAS %lu bytes now in DER is %lu bytes\n", buflen, generatedDERSize);
```

Comment:
This change seems to belong in a different commit, perhaps the previous as well.